### PR TITLE
Refs #29355 - report CVE identifiers

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
Do not merge until the linked PR is merged

**Comment by ares** [comment]
The linked PR is merged, good to go

**Comment by ares** [comment]
rebase and the metadata updated, the linked patch made it to katello 3.16, easy merge, anyone?

**Review by lzap — APPROVED** [review]
I don't see how I could break anything with this. :-)

**Pull request merged**
